src/components/course-settings/roster.cjsx

```diff
@@ -14,10 +14,27 @@ module.exports = React.createClass
   propTypes:
     courseId: React.PropTypes.string.isRequired
 
+  updateCourseName:(courseId)->
+    RosterActions.save(@props.courseId)
+    console.log('waka kinda working');
```

Comment:
`updateCourseName` probably needs to pull the value out of the `<input>` box. to do that you need to change it to be `<input ref='something'>` and then use `{something} = @refs` in here (and then `something.value()` to get the actual text.